datafusion/physical-expr/src/functions.rs

```diff
@@ -404,7 +404,7 @@ pub fn create_physical_fun(
             Arc::new(array_expressions::array_to_string)
         }
         BuiltinScalarFunction::Cardinality => Arc::new(array_expressions::cardinality),
-        BuiltinScalarFunction::MakeArray => Arc::new(array_expressions::array),
+        BuiltinScalarFunction::MakeArray => Arc::new(array_expressions::array_make),
```

Comment:
Won't `make_array` be a better, more consistent renaming for the function?

Reply:
I fixed this in d91f66704 (it was a little tricky because `make_array` is also a function in arrow so some new aliasing was needed